include/psa/crypto_types.h

```diff
@@ -115,8 +115,9 @@ typedef uint32_t psa_algorithm_t;
  *   whether the key is _volatile_ or _persistent_.
  *   See ::psa_key_persistence_t for more information.
  * - Bits 8-31 (#PSA_KEY_LIFETIME_GET_LOCATION(\c lifetime)):
- *   location indicator. This value indicates where the key is stored
- *   and where operations on the key are performed.
+ *   location indicator. This value indicates where the key material is stored
+ *   (or at least where it is accessible in cleartext) and where operations
```

Comment:
Why not just: "This value indicates where operations on the key are performed." or "This value indicates where operations using the key are performed." as eventually where it is stored is not completely relevant? It would probably be beneficial to explain why where it is stored is not completely relevant.

Reply:
It really is about who has access to the plaintext, and occasionally where wrapped material is stored. Where the operations take place is actually secondary. I've rephrased this passage.